### ISSUE TITLE
allow import types exported by ng-redux.d.ts 

### ISSUE THREAD

**Issue by jcperez-ch** [angular-redux/store#360]
I am trying to define a function selector in my app, but I want to be able to type it . accordingly like:

```js
import { FunctionSelector, NgRedux } from '@angular-redux/store';

export const mySelector: FunctionSelector<IAppState, boolean> = ({ user }) => user !== null;
```

But the type `FunctionSelector` is not exported inside  '@angular-redux/store'

It would be really handy to allow users to reuse those type definitions.

**Comment by e-schultz** [comment]
Shouldn't be to hard to add this, will try and address it soon.

**Comment by SethDavenport** [comment]
Yup I like this idea.

**Comment by SethDavenport** [comment]
Coming in 6.2, soon.

**Comment by SethDavenport** [comment]
This has been implemented in @angular-redux/store 6.2.0. Let me know if you run into any issues.

**Comment by jcperez-ch** [comment]
Thanks guys! I'll try it out
